applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -73,32 +71,35 @@ static const uint8_t cap_adv_data[] = {
 	BT_AUDIO_UNICAST_ANNOUNCEMENT_TARGETED,
 };
 
-#define AVAILABLE_SOURCE_CONTEXT                                                                   \
-	(BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED | BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL)
+#if CONFIG_BT_PACS_SNK_CONTEXT
+#define AVAILABLE_SINK_CONTEXT (CONFIG_BT_PACS_SNK_CONTEXT)
+#else
+#define AVAILABLE_SINK_CONTEXT BT_AUDIO_CONTEXT_TYPE_PROHIBITED
+#endif
 
-#define AVAILABLE_SINK_CONTEXT                                                                     \
-	(BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED | BT_AUDIO_CONTEXT_TYPE_MEDIA |                         \
-	 BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL)
+#if CONFIG_BT_PACS_SRC_CONTEXT
```

Comment:
So this means that CONFIG_BT_PACS_SRC_CONTEXT is not set if !STREAM_BIDIRECTIONAL?
![image](https://github.com/alexsven/sdk-nrf/assets/89020256/6dbe05ec-1ed8-4b3d-bea2-45496252c64b)

Reply:
I see that this config is removed entirely from the latest Zephyr version :/

Reply:
Need some work here, Alexander will change.

Reply:
Done